src/main/java/by/itech/lab/supplier/repository/UserRepository.java

```diff
@@ -31,4 +35,11 @@
 
     Page<User> findAllByActiveEquals(Pageable pageable, Boolean active);
 
+    @Modifying
+    @Query("update User set warehouse = :warehouse where id IN :usersId")
```

Comment:
please dont use different case for operators, if you use uppercase - so use it everywhere, the same for lower case

Reply:
thanks